clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -135,6 +168,12 @@ export const userAgentConfig = isBrowser
         async load() {
           try {
             const fileContent = await fs.readFile(this.filepath, "utf8");
+            // If the config file is the old template, and user has not modified it,
+            // Overwrite it with the new template.
+            if (fileContent.trim() === oldConfigTomlTemplate.trim()) {
+              await this.createTemplate();
+              return await this.load();
+            }
```

Comment:
Changes in the template does not take effect when user update tabby from old version and the `config.toml` file already exists.  
If user has not modified old template `config.toml`, we can overwrite it. Otherwise, we can not update it directly.